test/unit/com/unifina/service/StreamServiceSpec.groovy

```diff
@@ -160,7 +162,6 @@ class StreamServiceSpec extends Specification {
 		service.getReadAuthorizedStream("streamId", user, null, cb)
 		then:
 		thrown(NotPermittedException)
-		1 * service.permissionService.check(user, stream, Permission.Operation.STREAM_GET) >> false
```

Comment:
This should be invoked. See earlier comment about that one (accidentally?) removed line.